src/Backend/Modules/Tags/Ajax/Edit.php

```diff
@@ -45,15 +45,18 @@ public function execute()
                 if (BackendTagsModel::existsTag($tag)) {
                     $this->output(self::BAD_REQUEST, null, BL::err('TagAlreadyExists'));
                 } else {
-                    $item['id'] = $id;
-                    $item['tag'] = \SpoonFilter::htmlspecialchars($tag);
-                    $item['url'] = BackendTagsModel::getURL(
-                        CommonUri::getUrl(\SpoonFilter::htmlspecialcharsDecode($item['tag'])),
+                    $item = BackendTagsModel::get($id);
```

Comment:
I'd call this variable `$tag`, not `$item`. I know there are a lot of bad variable names in Fork (item, record, value, ...), and I'd really like to get these out. Adding new once is a no go for me :)